pandas/io/parquet.py

```diff
@@ -185,17 +188,14 @@ def write(
             )
 
     def read(self, path, columns=None, **kwargs):
-        if is_s3_url(path):
-            from pandas.io.s3 import get_file_and_filesystem
+        if is_fsspec_url(path):
+            import fsspec
 
-            # When path is s3:// an S3File is returned.
-            # We need to retain the original path(str) while also
-            # pass the S3File().open function to fastparquet impl.
-            s3, filesystem = get_file_and_filesystem(path)
-            try:
-                parquet_file = self.api.ParquetFile(path, open_with=filesystem.open)
-            finally:
-                s3.close()
+            # if path is s3:// or gs:// we need to open the file in 'wb' mode.
+            # TODO: Support 'ab'
```

Comment:
Bad copy-paste on this comment?

Reply:
This was left over from previous code and no longer relevant (it shows up because of changed indentation).